packages/wonder-blocks-dropdown/components/dropdown.js

```diff
@@ -26,10 +27,15 @@ type Props = {|
     children: (eventState: ClickableState) => React.Element<any>,
 
     /**
-     * Can be used to override the state of the Dropdown by parent elemnents
+     * Can be used to override the state of the Dropdown by parent elements
      */
     opened?: boolean,
 
+    /**
+     * Called when the menu closes
+     */
+    onClose?: () => mixed,
```

Comment:
Added this new `onClose` prop here to update the controlled component from its parent. Initially I tried to use the existing `onChange` prop, but found some issues when trying to use the some keyboard keys (`esc`, `enter`) and when clicking outside the menu (`document.mouseout`). Also, this feels more appropriate b/c if we close the menu by clicking outside, that means the dropdown hasn't changed any selection.